lib/python/test/setups/ESKHI/Data.py

```diff
@@ -0,0 +1,91 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+
+This test enables the ESKHI to be checked. It is irrelevant whether
+it is a 2D or 3D simulation. In a 3D simulation, however, attention
+must be paid to the Lorentz factor. Furthermore, the error in the
+growth rate must be taken into account in the 2D simulation (for more
+information, for example, see the bachelor thesis submitted
+by Mika Soren Voss).
+
+For information:
+    according to the documentation, the case distinction will only
+    be incorporated in a later version, which means that two case
+    s of KHI currently have to be treated differently.
+"""
+
+# flake8: noqa
+import os
+import sys
+
+sys.path.insert(1, os.path.dirname(os.path.abspath(__file__)) + "/../../")
+from testsuite import Math as tm
+
+# general information about the test
+title = "KHI Growthrate (2D ESKHI)"
+author = "Mika Soren Voss"
+
+
+# directory specified
+resultDirection = None
+paramDirection = "/bigdata/hplsim/scratch/voss29/myFirst_PIConGPU_runs/runs/KHI/002_KHI/input/include/picongpu/param"
+dataDirection = "/bigdata/hplsim/scratch/voss29/myFirst_PIConGPU_runs/runs/KHI/002_KHI/simOutput"
```

Comment:
This are very specific directories
perhaps something like:
 ```suggestion
 pathToDirs = "/pleae/add/your/directory/"
paramDirection = pathToDirs + "/runs/KHI/002_KHI/input/include/picongpu/param"
dataDirection = pathToDirs + "/runs/KHI/002_KHI/simOutput"
```
would be better